lib/inspec/resources/ibmdb2_conf.rb

```diff
@@ -0,0 +1,49 @@
+require "inspec/resources/ibmdb2_conf"
```

Comment:
Seems like this is the self required in this resource? Do we need this?

Reply:
Thanks. That should not be there.